src/constants/index.ts

```diff
@@ -2,8 +2,11 @@
  * TFS centralised constants, referred as `@ukef/constants`.
  * Following constants are served:
  *
- * 1. Application
+ * 1. acbs
+ * 2. Application
+ * 3. Auth
  */
 
 export * from './acbs';
 export * from './application';
+export * from './auth.constant';
```

Comment:
Should we have rename the other files to `acbs.constant.ts` and `application.constant.ts`?

Reply:
good spot!